MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,10 +1,144 @@
+        QuizQuestion(image: "The Godfather",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: true),
+        QuizQuestion(image: "The Dark Knight",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: true),
+        QuizQuestion(image: "Kill Bill",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: true),
+        QuizQuestion(image: "The Avengers",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: true),
+        QuizQuestion(image: "Deadpool",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: true),
+        QuizQuestion(image: "The Green Knight",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: true),
+        QuizQuestion(image: "Old",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: false),
+        QuizQuestion(image: "The Ice Age Adventures of Buck Wild",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: false),
+        QuizQuestion(image: "Tesla",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: false),
+        QuizQuestion(image: "Vivarium",
+                     text: "Рейтинг этого фильма больше чем 6?",
+                     correctAnswer: false)
+    ]
+    
+    
     // MARK: - Lifecycle
+    @IBOutlet weak private var imageView: UIImageView!
+    @IBOutlet weak private var textLabel: UILabel!
+    @IBOutlet weak private var counterLabel: UILabel!
+    private var currentQuestionIndex = 0
+    private var correctAnswers = 0
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        show(quiz: currentQuestionIndex)
+
+    }
+    
+    private func show(quiz questinIndex: Int) {
+        let currentQuestion = questions[currentQuestionIndex]
+        let quizStepViewModel = convert(model: currentQuestion)
+        imageView.image = quizStepViewModel.image
+        textLabel.text = quizStepViewModel.question
+        counterLabel.text = quizStepViewModel.questionNumber
+        
+    }
+    
+    private func showResult() {
+        let alert = UIAlertController(title: "Этот раунд окончен",
+                                      message: "Ваш результат: " + "\(correctAnswers)/\(questions.count)",
+                                      preferredStyle: .alert)
+        let action = UIAlertAction(title: "Сыграть еще раз", style: .default, handler: { _ in
+            self.currentQuestionIndex = 0
+            self.correctAnswers = 0
+            self.show(quiz: self.currentQuestionIndex)
+            
+        })
+    
+        alert.addAction(action)
+
+        self.present(alert, animated: true, completion: nil)
```

Comment:
Всегда используй наиболее короткую форму синтаксиса
```suggestion
        present(alert, animated: true, completion: nil)
```